### PR TITLE
Flex helper classes

### LINKED ISSUE #283: update CSS to appropriately render button groups/buttons on different screen sizes
This is a continuation of a conversation on the new redesign https://github.com/ember-learn/ember-website/pull/555#discussion_r378180714

essentially we are using margins on buttons on the homepage but when they wrap they are no longer centred properly.

We will probably need some way to put buttons into a "button row" or "button group" that deals with the behaviour correctly 👍

### PR DESCRIPTION
**Pull request by MinThaMie**
Fixes #283 

Adds flex-helper classes and documented the already present one

Screenshot 2022-07-18 at 12 01 39

### CODE REVIEW

File: docs/components/buttongroup.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Button Group
```
Comment (jenweber) [inline docs/components/buttongroup.md:1]: I think the filename here should probably be `button-group` in order to be consistent in the nav menu with other two word components, like link-card

**Comment by MelSumner** [comment]
For consideration: for links that look like buttons, adding support for the SPACE key can really help keyboard-only users.

**Comment by jenweber** [comment]
A caveat with these comments is that I have not been very involved with the website work/styleguide lately. But here are my thoughts!

-  _Should is be a more generic wrapper?_ No, I think it's fine for it to have one job
-   _The name button-group might be confusing for other people because it often means (e.g. Bootstrap) that the buttons will touch each other._ I like the name you chose. I have seen this same name in other component frameworks besides bootstrap, with the buttons not touching  
-   _Does it need the bottom margin?_ No, because when we use these components in the apps, we can use the spacing helpers. See https://deploy-preview-423--ember-styleguide.netlify.app/css/helpers/#spacinghelpers
-  _If we allow the @centered property do we want to change it to something that would support all the flex-alignments? Or is this something we could figure out somewhere else._ I suggest looking at the use cases we need to support. Probably we don't need anything fancy and centered will be enough. But I'm not sure.

Before merging, I recommend removing the margin and renaming that one file. Otherwise this looks good to go, from my perspective.

**Comment by jenweber** [comment]
P.S. to just say it directly, I think my review alone is probably not enough and someone who has been more active in this project should also take a look!

**Comment by MinThaMie** [comment]
> For consideration: for links that look like buttons, adding support for the SPACE key can really help keyboard-only users.

I think this should be part of a separate issue that actually does something with buttons instead of putting it in this one with the wrapper Classe

File: docs/components/buttongroup.md (hunk 1 above)
Comment (MinThaMie) [reply docs/components/buttongroup.md:1]: You're correct :)

**Review by jaredgalanis — APPROVED** [review]
This looks good to me 👍

**Review by mansona — CHANGES_REQUESTED** [review]
Hey folks sorry about not getting back to y'all on this one. We discussed this in the website redesign meeting and essentially it would make more sense that this was implemented as a CSS helper rather than a component. One of the philosophies of this new redesign implementation is that we should use the platform as much as possible, and when I discussed this with @MinThaMie it's clear that converting this to a CSS helper would even make it more useful in other places 🎉

**Review by mansona — APPROVED** [review]
This looks great 😍 thanks for making this change! much better 🎉

**Pull request merged**